cypress-tests/cypress/e2e/admin/pageBuilder/category/pageCategoriesSearchAndSort.cy.ts

```diff
@@ -0,0 +1,151 @@
+        cy.contains(categoryData4.name && categoryData4.url).should("exist");
+
+        //Assert all the categories are being displayed properly when searched for by name
+        cy.findByPlaceholderText("Search categories").clear().type(categoryData1.name);
+        cy.contains(categoryData1.name).should("exist");
+        cy.contains(categoryData2.name).should("not.exist");
+        cy.contains(categoryData3.name).should("not.exist");
+        cy.contains(categoryData4.name).should("not.exist");
+
+        cy.findByPlaceholderText("Search categories").clear().type(categoryData2.name);
+        cy.contains(categoryData1.name).should("not.exist");
+        cy.contains(categoryData2.name).should("exist");
+        cy.contains(categoryData3.name).should("not.exist");
+        cy.contains(categoryData4.name).should("not.exist");
+
+        cy.findByPlaceholderText("Search categories").clear().type(categoryData3.name);
+        cy.contains(categoryData1.name).should("not.exist");
+        cy.contains(categoryData2.name).should("not.exist");
+        cy.contains(categoryData3.name).should("exist");
+        cy.contains(categoryData4.name).should("not.exist");
+
+        cy.findByPlaceholderText("Search categories").clear().type(categoryData4.name);
+        cy.contains(categoryData1.name).should("not.exist");
+        cy.contains(categoryData2.name).should("not.exist");
+        cy.contains(categoryData3.name).should("not.exist");
+        cy.contains(categoryData4.name).should("exist");
+
+        //Assert all the categories are being displayed properly when searched for by URL
+        cy.findByPlaceholderText("Search categories").clear().type(categoryData1.url);
+        cy.contains(categoryData1.url).should("exist");
+        cy.contains(categoryData2.url).should("not.exist");
+        cy.contains(categoryData3.url).should("not.exist");
+        cy.contains(categoryData4.url).should("not.exist");
+
+        cy.findByPlaceholderText("Search categories").clear().type(categoryData2.url);
+        cy.contains(categoryData1.url).should("not.exist");
+        cy.contains(categoryData2.url).should("exist");
+        cy.contains(categoryData3.url).should("not.exist");
+        cy.contains(categoryData4.url).should("not.exist");
+
+        cy.findByPlaceholderText("Search categories").clear().type(categoryData3.url);
+        cy.contains(categoryData1.url).should("not.exist");
+        cy.contains(categoryData2.url).should("not.exist");
+        cy.contains(categoryData3.url).should("exist");
+        cy.contains(categoryData4.url).should("not.exist");
+
+        cy.findByPlaceholderText("Search categories").clear().type(categoryData4.url);
+        cy.contains(categoryData1.url).should("not.exist");
+        cy.contains(categoryData2.url).should("not.exist");
+        cy.contains(categoryData3.url).should("not.exist");
+        cy.contains(categoryData4.url).should("exist");
+
+        cy.findByPlaceholderText("Search categories")
```

Comment:
Loving `findByPlaceholderText`, hopefully we'll be able to make `findByText` work in the near future 🙏🏻